### PR TITLE
{lang} [intel/2017a] PyCUDA/2016.1.2 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic155 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/f2941ae52db1a0b118100b31a30421bd for a full test report.

File: easybuild/easyconfigs/p/PyCUDA/PyCUDA-2016.1.2-intel-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('CUDA', '8.0.61', '', True),
+    ('Boost', '1.63.0', versionsuffix),
+]
+
+prebuildopts = "./configure.py --cuda-root=$EBROOTCUDA --boost-inc-dir=$EBROOTBOOST/include/boost/ --boost-lib-dir=$EBROOTBOOST/lib/ --no-use-shipped-boost --boost-python-libname=boost_python && "
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyCUDA/PyCUDA-2016.1.2-intel-2017a-Python-2.7.13.eb]: @wpoely86 please break this up, line is (way) too long...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/948bb1319bbcd083daf23d9aeaf71b54 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2571.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c7704fc058eb89b12868717b3578d4f4 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**